### ISSUE TITLE
Not import ComputeDataComputeShader.compute

### ISSUE THREAD

**Issue by aftabali40** [raphael-ernaelsten/Aura#88]
**Not import ComputeDataComputeShader.compute**

**Screenshots**
![pic 1](https://user-images.githubusercontent.com/19773546/49072507-32e66480-f252-11e8-8bfe-0bd972d96dd1.jpg)
![pic 2](https://user-images.githubusercontent.com/19773546/49072508-32e66480-f252-11e8-955a-27b509fb1292.jpg)

**System (please complete the following information):**
 - OS: [e.g. Windows 10 Home]
 - Unity version: [e.g. 2018.3.0b1]
 - Graphic card: [e.g. HD Radeon 8670m]
 - Processor: [e.g. Intel i5 4200U]
 - RAM: [e.g. 8GB]

**Comment by kingbaggot** [comment]
Yup - I've got this and it does eventually compile (after about 40 mins - this is mentioned in the readme) 
However once it's completed I still get the same nullReferenceException (and untickable Aura component) as the people who quit out before it has compiled.

Have tried reimporting this 2-3 times on 2017.3 and 2018.2 and get the same errors.

MP Z4 G4 Workstation
Intel core i9-7940X
Ram 32 gb

really hope we can all get this working, as it looks fantastic !

**Comment by kingbaggot** [comment]
I have it working now - I stopped trying to add it to my current project and downloaded this persons unitypackage (linked below) and followed the steps.  
https://lmhpoly.com/unity-tutorial-volumetric-lighting/

It's VERY VERY good this asset. Well done Raphael !

will now try getting it into my own proj.

**Comment by xiangxiangjiang** [comment]
i have the same problem. my unity version is 2018.03. whether is it a bug when import this assets ?  Thanks for the assets.
